src/test/mir-opt/instrument_coverage.main.InstrumentCoverage.diff

```diff
@@ -1,51 +0,0 @@
-+ // MIR for `main` after InstrumentCoverage
-  
-  fn main() -> () {
-      let mut _0: ();                      // return place in scope 0 at /the/src/instrument_coverage.rs:10:11: 10:11
-      let mut _1: ();                      // in scope 0 at /the/src/instrument_coverage.rs:10:1: 16:2
-      let mut _2: bool;                    // in scope 0 at /the/src/instrument_coverage.rs:12:12: 12:17
-      let mut _3: !;                       // in scope 0 at /the/src/instrument_coverage.rs:12:18: 14:10
-  
-      bb0: {
-          falseUnwind -> [real: bb1, cleanup: bb6]; // scope 0 at /the/src/instrument_coverage.rs:11:5: 15:6
-      }
-  
-      bb1: {
-          StorageLive(_2);                 // scope 0 at /the/src/instrument_coverage.rs:12:12: 12:17
-          _2 = bar() -> [return: bb2, unwind: bb6]; // scope 0 at /the/src/instrument_coverage.rs:12:12: 12:17
-                                           // mir::Constant
-                                           // + span: /the/src/instrument_coverage.rs:12:12: 12:15
-                                           // + literal: Const { ty: fn() -> bool {bar}, val: Value(Scalar(<ZST>)) }
-      }
-  
-      bb2: {
-          FakeRead(ForMatchedPlace, _2);   // scope 0 at /the/src/instrument_coverage.rs:12:12: 12:17
-+         Coverage::Counter(1) for /the/src/instrument_coverage.rs:12:12 - 12:17; // scope 0 at /the/src/instrument_coverage.rs:12:9: 14:10
-          switchInt(_2) -> [false: bb4, otherwise: bb3]; // scope 0 at /the/src/instrument_coverage.rs:12:9: 14:10
-      }
-  
-      bb3: {
-          falseEdge -> [real: bb5, imaginary: bb4]; // scope 0 at /the/src/instrument_coverage.rs:12:9: 14:10
-      }
-  
-      bb4: {
-          _1 = const ();                   // scope 0 at /the/src/instrument_coverage.rs:12:9: 14:10
-          StorageDead(_2);                 // scope 0 at /the/src/instrument_coverage.rs:15:5: 15:6
-+         Coverage::Expression(4294967295) = 1 - 2 for /the/src/instrument_coverage.rs:15:6 - 15:7; // scope 0 at /the/src/instrument_coverage.rs:11:5: 15:6
-          goto -> bb0;                     // scope 0 at /the/src/instrument_coverage.rs:11:5: 15:6
-      }
-  
-      bb5: {
-          _0 = const ();                   // scope 0 at /the/src/instrument_coverage.rs:13:13: 13:18
-          StorageDead(_2);                 // scope 0 at /the/src/instrument_coverage.rs:15:5: 15:6
-+         Coverage::Counter(2) for /the/src/instrument_coverage.rs:13:13 - 13:18; // scope 0 at /the/src/instrument_coverage.rs:16:2: 16:2
-+         Coverage::Expression(4294967294) = 2 + 0 for /the/src/instrument_coverage.rs:16:1 - 16:2; // scope 0 at /the/src/instrument_coverage.rs:16:2: 16:2
-          return;                          // scope 0 at /the/src/instrument_coverage.rs:16:2: 16:2
-      }
-  
-      bb6 (cleanup): {
-          resume;                          // scope 0 at /the/src/instrument_coverage.rs:10:1: 16:2
-      }
-  }
-  
```

Comment:
I renamed a test, but since bless does not cleanup already created diffs I ran a script myself to delete all diff files, and then ran bless again. Since ci passes, this file is not used in any test afaict.